tools/generatemodel.py

```diff
@@ -202,11 +250,23 @@ def generate_enum(self, class_name, literals_path, sorter=None):
 
 
 if __name__ == '__main__':
-    # extractor = MetadataExtractor("./test.yaml")
-    version_tag = 'v1.8.0'
-    module_version_tag = version_tag.replace('.', '_')
-    generator = ClassGenerator("../schema/model_base.yaml", f"../schema/schema_{version_tag}.json", module_version_tag)
-    generator.generate(version_tag)
+    # Fetch the latest schema version and URL
```

Comment:
re-introduce arg parser to allow providing a specific schema version to be generated